src/aics-image-viewer/components/ChannelsWidgetRow/index.tsx

```diff
@@ -177,52 +161,45 @@ const ChannelsWidgetRow: React.FC<ChannelsWidgetRowProps> = (props: ChannelsWidg
     </Col>
   );
 
-  const renderControls = (): React.ReactNode => (
-    <div style={STYLES.settingsContainer}>
-      {volumeChecked && (
-        <Row justify="space-between" className="volume-settings">
-          <h4 className="ant-list-item-meta-title">Volume settings:</h4>
-          {createTFEditor()}
-        </Row>
-      )}
-      {isosurfaceChecked && <Row justify="space-between">{renderSurfaceControls()}</Row>}
-    </div>
-  );
+  const renderControls = (): React.ReactNode => {
+    if (!volumeChecked && !isosurfaceChecked) {
+      return <h4 style={{ fontStyle: "italic" }}>Not currently visible</h4>;
+    }
+    return (
+      <>
+        {volumeChecked && (
+          <>
+            <h4>Volume settings:</h4>
+            {createTFEditor()}
+          </>
+        )}
+        {isosurfaceChecked && (
+          <>
+            <h4>Surface settings:</h4>
+            {renderSurfaceControls()}
+          </>
+        )}
+      </>
+    );
+  };
 
-  const rowClass = controlsOpen ? "row-card" : "row-card controls-closed";
+  const rowClass = controlsOpen ? "channel-row" : "channel-row controls-closed";
   return (
-    <List.Item key={index} className={rowClass} extra={renderActions()}>
+    <List.Item key={index} className={rowClass}>
       <List.Item.Meta title={<span style={STYLES.channelName}>{props.name}</span>} avatar={createColorPicker()} />
-      {controlsOpen && renderControls()}
+      {visibilityControls}
+      {controlsOpen && <div style={{ width: "100%" }}>{renderControls()}</div>}
```

Comment:
I like how clean this is, but have been guided away from inline styling in the past.

This is not a request for change, but just curious your thought process in deciding to use inline here.

Reply:
This codebase is on the older side and some components have an odd pattern of doing styling with big JS objects laid out like stylesheets at the bottom of .tsx files. (I ended up having to add a type for these objects, `Styles`, when I converted the project to TS.) Though I don't take a super hard line on inline styling, I agree it's not great practice and am not really pleased to be doing it here. I just didn't try to break this file all the way out of that pattern... for now. I think this particular case was a sort of half-measure for a style record that I was able to trim down pretty far but not completely get rid of.